Pod/Classes/URBNAlertViewController.m

```diff
@@ -16,6 +16,14 @@
 #import <URBNConvenience/URBNTextField.h>
 #import "URBNAlertAction.h"
 
+@interface URBNAlertController(Private)
+
+- (void)dismissingAlert;
+- (void)addAlertToQueueWithAlertViewController:(URBNAlertViewController *)avc;
```

Comment:
I am using private methods from URBNAlertController here-- I am just exposing them via this category. I wanted to keep the AlertController.h file clear up private stuff